src/wallet/rpc/wallet.cpp

```diff
@@ -59,6 +59,7 @@ static RPCHelpMan getwalletinfo()
                         }, /*skip_type_check=*/true},
                         {RPCResult::Type::BOOL, "descriptors", "whether this wallet uses descriptors for scriptPubKey management"},
                         {RPCResult::Type::BOOL, "external_signer", "whether this wallet is configured to use an external signer such as a hardware wallet"},
+                        {RPCResult::Type::BOOL, "blank", "Whether this wallet is marked as intentionally not having any keys, scripts, or descriptors"},
```

Comment:
Using “not” in the context of a boolean feels always a bit like a double negation. Perhaps go with “without”:

```suggestion
                        {RPCResult::Type::BOOL, "blank", "whether this wallet was intentionally created without keys, scripts, or descriptors"},
```

Reply:
Changed the wording slightly, although did not use this suggestion since it is incorrect.